Doc/whatsnew/3.13.rst

```diff
@@ -650,583 +654,669 @@ ast
+  :manpage:`timer file descriptors <timerfd_create(2)>`
+  via :func:`~os.timerfd_create`,
+  :func:`~os.timerfd_settime`, :func:`~os.timerfd_settime_ns`,
+  :func:`~os.timerfd_gettime`, :func:`~os.timerfd_gettime_ns`,
+  :const:`~os.TFD_NONBLOCK`, :const:`~os.TFD_CLOEXEC`,
+  :const:`~os.TFD_TIMER_ABSTIME`, and :const:`~os.TFD_TIMER_CANCEL_ON_SET`
+  (Contributed by Masaru Tsuchiyama in :gh:`108277`.)
+
+* :func:`~os.lchmod` and the *follow_symlinks* argument of :func:`~os.chmod`
+  are both now available on Windows.
+  Note that the default value of *follow_symlinks*
+  in :func:`!lchmod` is ``False`` on Windows.
   (Contributed by Serhiy Storchaka in :gh:`59616`.)
 
-* Add support of :func:`os.fchmod` and a file descriptor
-  in :func:`os.chmod` on Windows.
+* :func:`~os.fchmod` and support for file descriptors in :func:`~os.chmod`
+  are both now available on Windows.
   (Contributed by Serhiy Storchaka in :gh:`113191`.)
 
-* :func:`os.posix_spawn` now accepts ``env=None``, which makes the newly spawned
-  process use the current process environment.
+* On Windows, :func:`~os.mkdir` and :func:`~os.makedirs` now support passing
+  a *mode* value of ``0o700`` to apply access control to the new directory.
+  This implicitly affects :func:`tempfile.mkdtemp`
+  and is a mitigation for :cve:`2024-4030`.
+  Other values for *mode* continue to be ignored.
+  (Contributed by Steve Dower in :gh:`118486`.)
+
+* :func:`~os.posix_spawn` now accepts ``None`` for the *env* argument,
+  which makes the newly spawned process use the current process environment.
   (Contributed by Jakub Kulik in :gh:`113119`.)
 
-* :func:`os.posix_spawn` gains an :attr:`os.POSIX_SPAWN_CLOSEFROM` attribute for
-  use in ``file_actions=`` on platforms that support
+* :func:`~os.posix_spawn` can now use the :attr:`~os.POSIX_SPAWN_CLOSEFROM`
+  attribute in the *file_actions* parameter on platforms that support
   :c:func:`!posix_spawn_file_actions_addclosefrom_np`.
   (Contributed by Jakub Kulik in :gh:`113117`.)
 
-* :func:`os.mkdir` and :func:`os.makedirs` on Windows now support passing a
-  *mode* value of ``0o700`` to apply access control to the new directory. This
-  implicitly affects :func:`tempfile.mkdtemp` and is a mitigation for
-  :cve:`2024-4030`. Other values for *mode* continue to be ignored.
-  (Contributed by Steve Dower in :gh:`118486`.)
 
-os.path
--------
+:mod:`os.path`
+--------------
 
-* Add :func:`os.path.isreserved` to check if a path is reserved on the current
-  system. This function is only available on Windows.
+* Add :func:`~os.path.isreserved` to check if a path is reserved
+  on the current system.
+  This function is only available on Windows.
   (Contributed by Barney Gale in :gh:`88569`.)
 
-* On Windows, :func:`os.path.isabs` no longer considers paths starting with
-  exactly one (back)slash to be absolute.
+* On Windows, :func:`~os.path.isabs` no longer considers paths
+  starting with exactly one (back)slash (``\`` or ``/``) to be absolute.
```

Comment:
Shall we just say "slash"? We show the two types to make it clear:
```suggestion
  starting with exactly one slash (``\`` or ``/``) to be absolute.
```

(Or more explicit:)

```suggestion
  starting with exactly one forward or backslash (``\`` or ``/``) to be absolute.
```